osu.ElasticIndexer/Commands/Index/NukeAllIndicesCommand.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Threading;
+using McMaster.Extensions.CommandLineUtils;
+using Nest;
+using osu.Server.QueueProcessor;
+
+namespace osu.ElasticIndexer.Commands.Index;
+
+[Command("nuke", Description = "Resets everything with fire.")]
+public class NukeAllIndicesCommand : ListIndicesCommand
```

Comment:
this looks fun if you have the non-score indices on the same instance for dev 👀

Reply:
It should probably list the indices it's going to be deleting at the start as well (to make it obvious if you're deleting more than you expected)?

There's also a way to delete using `scores_*` but that needs the `action.destructive_requires_name` setting to be `false` and es8+ defaults to `true`

Reply:
> It should probably list the indices it's going to be deleting at the start as well (to make it obvious if you're deleting more than you expected)?

It already does this 👍